lib/types/types.d.ts

```diff
@@ -168,6 +168,7 @@ declare global {
         device_options: KeyValue;
         advanced: {
             log_rotation: boolean;
+            log_console_json: boolean;
```

Comment:
```suggestion
            log_fromat: 'json' | 'text';
```

I think this is more extendable in the future

Reply:
Shouldn't that be log_format_console if you want to do it that way? The file logs are a different category.

Reply:
Just had a chat with @Nerivec, let's keep it as is, could you make a PR for the docs then I can merge this.

Reply:
I believe in general that apps should not spend a huge amount of work for multiple logging destinations and various formatting options. That is nowadays mostly the task of the deployment infrastructure, for example systemd or docker, and they do a decent job at that. 

If you want a big refactoring of the current logging, you would have to order all the options on a per channel basis, for example having a file channel, a syslog channel, a console channel each having a format and extra channel specific options. I think this is going overboard and not worth the effort.

Reply:
@andrejsoucek because it is unlikely that there will be any other formats than the existing one or json.